README.md

```diff
@@ -29,57 +54,55 @@ points in time.
 This extension hooks into the SchemaTool generation process so that it will automatically
 create the necessary DDL statements for your audited entities.
 
-## Installation (In Symfony2 Application)
+## Installation
 
 ### Installing the bundle
 
-Simply run assuming you have installed composer.phar or composer binary:
+Simply run assuming you have composer:
 
-``` bash
-$ php composer.phar require simplethings/entity-audit-bundle
+```bash
+$ composer require sonata-project/entity-audit-bundle
 ```
 
 ### Enable the bundle
 
 Finally, enable the bundle in the kernel:
 
-``` php
-// app/AppKernel.php
+```php
+// config/bundles.php
 
-public function registerBundles()
-{
-    $bundles = array(
-        //...
-        new SimpleThings\EntityAudit\SimpleThingsEntityAuditBundle(),
-        //...
-    );
-    return $bundles;
-}
+return [
+    //...
+    SimpleThings\EntityAudit\SimpleThingsEntityAuditBundle::class => ['all' => true],
+    //...
+];
 ```
 
 ### Configuration
 
-Load extension "simple_things_entity_audit" and specify the audited entities (yes, that ugly for now!)
+Load extension "simple_things_entity_audit" and specify the audited entities
+
+```yaml
+# config/packages/entity_audit.yml
 
-##### app/config/config.yml
-```yml
 simple_things_entity_audit:
     audited_entities:
         - MyBundle\Entity\MyEntity
         - MyBundle\Entity\MyEntity2
 ```
 If you need to exclude some entity properties from triggering a revision use:
 
-##### app/config/config.yml
-```yml
+```yaml
+# config/packages/entity_audit.yml
```

Comment:
IMO, we could directly enforce the `.yaml` extension.
```suggestion
# config/packages/entity_audit.yaml
```

Reply:
Indeed, done